examples/AutoConnectWithFSParameters/AutoConnectWithFSParameters.ino

```diff
@@ -33,39 +33,42 @@ void setup() {
   //SPIFFS.format();
 
   //read configuration from FS json
-  Serial.println("mounting FS...");
+  Serial.println(F("mounting FS..."));
 
   if (SPIFFS.begin()) {
-    Serial.println("mounted file system");
+    Serial.println(F("mounted file system"));
     if (SPIFFS.exists("/config.json")) {
       //file exists, reading and loading
-      Serial.println("reading config file");
+      Serial.println(F("reading config file"));
       File configFile = SPIFFS.open("/config.json", "r");
       if (configFile) {
-        Serial.println("opened config file");
+        Serial.println(F("opened config file"));
         size_t size = configFile.size();
         // Allocate a buffer to store contents of the file.
         std::unique_ptr<char[]> buf(new char[size]);
 
         configFile.readBytes(buf.get(), size);
-        DynamicJsonBuffer jsonBuffer;
-        JsonObject& json = jsonBuffer.parseObject(buf.get());
-        json.printTo(Serial);
-        if (json.success()) {
-          Serial.println("\nparsed json");
-
-          strcpy(mqtt_server, json["mqtt_server"]);
-          strcpy(mqtt_port, json["mqtt_port"]);
-          strcpy(blynk_token, json["blynk_token"]);
-
+        StaticJsonDocument<500> jsonBuffer;
+        auto error = deserializeJson(jsonBuffer, buf.get());
```

Comment:
this declaration is for json 5 not 6.

Reply:
Nop, the declaration for 6 is like that, because in the last version of `ArduinoJson` for small documents we have:

```
You can choose to store your JsonDocument in the stack or in the heap:

Use a StaticJsonDocument to store in the stack (recommended for documents smaller than 1KB)
Use a DynamicJsonDocument to store in the heap (recommended for documents larger than 1KB)
```

I have a implementation updated of `AutoConnectWithFSParameters.ino` example, like that:

```java
StaticJsonDocument<512> json;
// Deserialize the JSON document
DeserializationError error = deserializeJson(json, configFile);
if (error)
    Serial.println(F("-WM: Failed to read config file, using default configuration"));
```

The current example is outdated, it is only for ArduinoJson 5.